.eslintrc

```diff
@@ -0,0 +1,52 @@
+{
+  "env": {
+    "amd": true,
+    "es6": true,
+    "jasmine": true,
+    "node": true,
+    "mocha": true,
+    "browser": true,
+    "builtin": true
+  },
+  "rules": {
+    "camelcase": 0,
+    "comma-dangle": 0,
+    "complexity": [2, 6],
+    "consistent-return": 0,
+    "curly": [2, "all"],
+    "dot-notation": 0,
+    "eqeqeq": [2, "allow-null"],
+    "global-strict": [0, "never"],
+    "globals": {
+      "define": true,
+      "Promise": true
```

Comment:
And I don't think we need to define "Promise" as a global if we're using the `es6` env.

Actually, here's my hacked version of your /.eslintrc file with modified "envs" and everything:
```json
{
  "env": {
    "amd": true,
    "es6": true,
    "mocha": true,
    "node": true
  },
  "rules": {
    "camelcase": 0,
    "comma-dangle": 0,
    "complexity": [2, 6],
    "consistent-return": 0,
    "curly": [2, "all"],
    "dot-notation": 0,
    "eqeqeq": [2, "allow-null"],
    "global-strict": [0, "never"],
    "handle-callback-err": 0,
    "key-spacing": 0,
    "new-cap": 0,
    "no-cond-assign": [2, "except-parens"],
    "no-console": 0,
    "no-debugger": 2,
    "no-empty": 0,
    "no-eval": 2,
    "no-irregular-whitespace": 2,
    "no-loop-func": 0,
    "no-multi-spaces": 0,
    "no-new": 2,
    "no-script-url": 2,
    "no-sequences": 2,
    "no-shadow": 0,
    "no-spaced-func": 0,
    "no-undef": 2,
    "no-underscore-dangle": 0,
    "no-unused-vars": [0, {"vars": "all", "args": "none"}],
    "no-use-before-define": true,
    "no-with": 2,
    "quotes": [2, "single", "avoid-escape"],
    "semi": [2, "always"],
    "space-unary-ops": 0,
    "strict": 2,
    "valid-typeof": 1,
    "wrap-iife": 0,
    "yoda": 0
  }
}
```

Feel free to give it a quick `diff` and see the changes. I think I removed some browser specific `env`s from the config and only specified them in the app/.eslintrc file, plus added the `no-console: 0`, and added that "avoid-escape" parameter on `quotes` rule.